newIDE/app/src/LocalApp.js

```diff
@@ -43,6 +40,10 @@ export const create = (authentication: Authentication) => {
   const appArguments = Window.getArguments();
   const isDev = Window.isDev();
 
+  // Search "activate cloud projects" in the codebase for everything to
+  // remove once cloud projects are activated for the desktop app.
+  const supportsCloudProjects = Window.isDev();
```

Comment:
why not remove this now?

Reply:
I think we can try this in development for a few days before enabling this in production?

Reply:
Resolving, we can always remove this in a subsequent PR if it works well.